week-4/E-arrays-of-objects/exercise-1.js

```diff
@@ -5,42 +5,59 @@
 */
 
 var person1 = {
-    name: "Alice",
-    age: 25
+  name: "Alice",
+  age: 25
 };
 
 var person2 = {
-    name: "Bob",
-    age: 30
+  name: "Bob",
+  age: 30
 };
 
 var person3 = {
-    name: "John",
-    age: 20
+  name: "John",
+  age: 20
 };
 
 /*
 DO NOT EDIT ANYTHING ABOVE THIS LINE
 WRITE YOUR CODE BELOW
 */
 
+var persons = [];
+persons.push(person1); // Complete here
+persons.push(person2);
+persons.push(person3);
+console.log(persons);
 
-var persons = // Complete here
-
-var personNames = // Complete here
-
-var personsYoungerThan28YearsOld = // Complete here
+var personNames = persons.map(element => element.name); // Complete here
```

Comment:
Maybe use `person` rather than `element` here, since you know it's a person. Also it's a good idea to remove unnecessary or irrelevant comments in code. So one you've done the work you can remove the `// Complete here` comment.